### ISSUE TITLE
Blast oyster genes to Uniprot

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#987]
query - https://d.pr/f/gTccOb

database - screenshot https://d.pr/i/jzcInD

see https://www.uniprot.org/downloads#uniprotkblink

use blastx note you can pick number of cpus based on your machine, below example is 4

`-evalue 1E-10 -num_threads 4 -max_target_seqs 1 -outfmt 6`

**Comment by meganewing** [comment]
[oysterblast.tab.zip](https://github.com/RobertsLab/resources/files/5092163/oysterblast.tab.zip)

**Comment by meganewing** [comment]
Okay. So I've hit a wall. 
The blast results didn't have a single column with _just_ the gene identifiers. So I created a csv file of just the gene IDs using sub str. I then uploaded that csv file to uniprot mapping and selected the expression categories we wanted (tissue specificity, induction, and developmental stage) and saved the uniprot table as a csv.

The issue is with the gene id csv file (shortGene) and the original blast (oysterBlast) results there were ~550 rows. With the uniprot table, there is only ~190 rows. Where did the rest of the rows go? 

I've attached all 3 files here in a zip. 
[oysterUniprotHelp.zip](https://github.com/RobertsLab/resources/files/5242766/oysterUniprotHelp.zip)
